.github/ISSUE_TEMPLATE/PROPOSAL_TEMPLATE.md

```diff
@@ -5,17 +5,24 @@ title: '[PROPOSAL]'
 labels: proposal
 assignees: ''
 ---
-**What kind of business use case are you trying to solve? What are your requirements?**
-A clear and concise description of the business problem, e.g. _My customers are asking for [...]_
 
-**What is the problem? What is preventing you from meeting the requirements?**
-Describe what impact this problem has for lack of a solution.
+## What are you proposing?
```

Comment:
If we are going to use `##`, let's change all templates (and maybe use `###`). Other templates use `**`. I do like `###` because it can be permalinked, but maybe it's not necessary within a single issue.

Reply:
I'm supportive of `###` - but I like semantic markup, especially since we can easily link it.